std/base64.d

```diff
@@ -104,6 +104,21 @@ alias Base64URL = Base64Impl!('-', '_');
     assert(Base64URL.decode("g9cwegE_") == data);
 }
 
+/**
+ * Unpadded variation of Base64 encoding that is safe for use in URLs and
+ * filenames, as used in RFCs 4648 and 7515 (JWS/JWT/JWE).
+ *
+ * See $(LREF Base64Impl) for a description of available methods.
+ */
+alias Base64URLNoPadding = Base64Impl!('-', '_', Base64.NoPadding);
+
+///
+@safe unittest
+{
+    ubyte[] data = [0x83, 0xd7, 0x30, 0x7b, 0xef];
+    assert(Base64URLNoPadding.encode(data) == "g9cwe-8");
+    assert(Base64URLNoPadding.decode("g9cwe-8") == data);
+}
 
 /**
  * Template for implementing Base64 encoding and decoding.
```

Comment:
I think the documentation of `Base64Impl` needs to be updated, because it currently says:

```
...
  * this module provides two default implementations: $(LREF Base64) and
  * $(LREF Base64URL), that implement basic Base64 encoding and a variant
  * intended for use in URLs and filenames, respectively.
```